### DISCUSSION TITLE
enable auto loading the the model on windows start

### DISCUSSION THREAD

**Discussion by master2142** [oobabooga/text-generation-webui#3177] [Ideas]
We should have a feature to enable auto loading the the model on windows start and and auto running the api with an auto  prompt so the llm will  follow orders during a reboot if we start making portable robots in the future or diy task's on a computer 🙂

**Comment by missionfloyd** [comment]
You can make it automatically load a model with the `--model` [argument](https://github.com/oobabooga/text-generation-webui#basic-settings).

To run it on startup, make a shortcut with your commandline and put it in the [startup folder](https://support.microsoft.com/en-us/windows/add-an-app-to-run-automatically-at-startup-in-windows-10-150da165-dcd9-7230-517b-cf3c295d89dd).

**Reply by master2142** [reply]
Thanks i'll try it out appreciate it.
